.gitignore

```diff
@@ -1,6 +1,7 @@
 ./doc/chap*
 ./examples/output/*
 ./tst/output/*
+./doc/chap*
```

Comment:
Isn't this the same as line 1?

Reply:
Strange, the generated files were listed as untracked as soon as I had processed the manual, and they disappear from the list of untracked files after my addition.
Even more strange, now the generated files are again listed as untracked; they disappear from this list when I remove the leading `./` from the lines in `.gitignore` --I do not understand what is going on here.

Reply:
The problem is that the .gitignore is simply wrong. Fixed in PR #11

Reply:
Thanks. The format from #11 works for me.